docs/rules/attribute-hyphenation.md

```diff
@@ -1,17 +1,17 @@
-# enforce attribute naming style in template (vue/attribute-hyphenation)
+# enforce attribute naming style on custom components in template (vue/attribute-hyphenation)
```

Comment:
This part is automatically generated based on the rule's metadata informations. If you want to update it please - do so in rule file instead. Otherwise it'll be overwritten during next release.

Reply:
Is it just this one line coming from `meta.docs.description` and needs updating? If so I've done it, let me know if there is anything else.